src/main/java/net/syscon/elite/api/model/v1/AvailableDates.java

```diff
@@ -15,6 +19,6 @@
 @ToString
 public class AvailableDates {
 
-    @ApiModelProperty(value = "Available Dates", allowEmptyValue = true)
+    @ApiModelProperty(value = "Available Dates", dataType = "[Ljava.sql.Date;", allowEmptyValue = true)
```

Comment:
had to manually supply data type as wasn't inferred correctly leaving #definitions/LocalDate not defined error